src/middleware.mjs

```diff
@@ -22,25 +25,30 @@ class Upload {
           this.done = true
```

Comment:
It looks like this is not used anywhere anymore?

Reply:
Yep, that's correct! Removed.